api/server/server.go

```diff
@@ -402,6 +610,18 @@ func (s *Server) startGears(ctx context.Context, cancel context.CancelFunc) {
 
 	installChildReaper()
 
+	if s.nodeType == ServerTypePureRunner {
+		// Run grpc too
+		grpcAddr := fmt.Sprintf("%s:%d", whoAmI().String(), s.grpcListenPort)
```

Comment:
is it possible to skip specifying local IP part? I think server (http.Server) listen and gRPC listen should be in sync with each other regarding this.. If we decide to bind to a specific local IP, then both servers should do the same.